static/src/javascripts/__flow__/types/ophan.js

```diff
@@ -54,7 +54,8 @@ declare type OphanComponentType =
     | 'ACQUISITIONS_EDITORIAL_LINK'
     | 'ACQUISITIONS_SUBSCRIPTIONS_BANNER'
     | 'ACQUISITIONS_OTHER'
-    | 'SIGN_IN_GATE';
+    | 'SIGN_IN_GATE'
+    | 'BRAZE_BANNER';
```

Comment:
Is creating a new component type as simple as defining it here? Does it need to be added somewhere else first?

Reply:
No, we need to add the new type to the Ophan Thrift definitions first (and update dependent projects).